go/runtime/registry/config.go

```diff
@@ -37,6 +37,9 @@ const (
 	CfgRuntimePaths = "runtime.paths"
 	// CfgSandboxBinary configures the runtime sandbox binary location.
 	CfgSandboxBinary = "runtime.sandbox.binary"
+	// CfgRuntimeEnv sets the runtime environment. Setting an environment that does not
```

Comment:
Looks like something is up with the linter, maybe since the update to Go 1.18?
```suggestion
	// CfgRuntimeEnvironment sets the runtime environment. Setting an environment that does not
```

Reply:
Okay, this is totally messed up. I'm going to fix the linter. Sorry for using so much `jernej-lint`. Next time won't have issues!